Error installing on Raspberry Pi (256mb)

Hey, I just tried to install the update for use with node-opus, but I'm having an issue:
```
pi@raspberrypi ~/nodeplayer $ node -v
v0.12.2
pi@raspberrypi ~/nodeplayer $ npm -v
2.7.4
pi@raspberrypi ~/nodeplayer $ npm install ogg@1.2.0
|
> ogg@1.2.0 install /home/pi/nodeplayer/node_modules/ogg
> node-gyp rebuild

make: Entering directory '/home/pi/nodeplayer/node_modules/ogg/build'
  CC(target) Release/obj.target/libogg/deps/libogg/src/framing.o
In file included from ../deps/libogg/include/ogg/ogg.h:25:0,
                 from ../deps/libogg/src/framing.c:25:
../deps/libogg/include/ogg/os_types.h:143:32: fatal error: ogg/config_types.h: No such file or directory
compilation terminated.
deps/libogg/libogg.target.mk:89: recipe for target 'Release/obj.target/libogg/deps/libogg/src/framing.o' failed
make: *** [Release/obj.target/libogg/deps/libogg/src/framing.o] Error 1
make: Leaving directory '/home/pi/nodeplayer/node_modules/ogg/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:269:23)
gyp ERR! stack     at ChildProcess.emit (events.js:110:17)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:1074:12)
gyp ERR! System Linux 3.18.12+
gyp ERR! command "node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /home/pi/nodeplayer/node_modules/ogg
gyp ERR! node -v v0.12.2
gyp ERR! node-gyp -v v1.0.3
gyp ERR! not ok 
npm ERR! Linux 3.18.12+
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "ogg@1.2.0"
npm ERR! node v0.12.2
npm ERR! npm  v2.7.4
npm ERR! code ELIFECYCLE

npm ERR! ogg@1.2.0 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the ogg@1.2.0 install script 'node-gyp rebuild'.
...
```

Any ideas?

--- a/README.md
+++ b/README.md
@@ -87,5 +87,7 @@ Please send pull requests for additional modules if you write one.
 | **Module**                       | **Decoder?** | **Encoder?**
 |:--------------------------------:|:------------:|:------------:
 |   [`node-vorbis`][node-vorbis]   |      ✓       |      ✓
+|   [`node-opus`][node-opus]       |      ✓       |      ✓
 
 [node-vorbis]: https://github.com/TooTallNate/node-vorbis
+[node-opus]: https://github.com/Rantanen/node-opus

--- a/deps/libogg/config/linux/arm/ogg/config.h
+++ b/deps/libogg/config/linux/arm/ogg/config.h
@@ -1,0 +1,102 @@
+/* config.h.  Generated from config.h.in by configure.  */
+/* config.h.in.  Generated from configure.in by autoheader.  */
+
+/* Define to 1 if you have the <dlfcn.h> header file. */
+#define HAVE_DLFCN_H 1
+
+/* Define to 1 if you have the <inttypes.h> header file. */
+#define HAVE_INTTYPES_H 1
+
+/* Define to 1 if you have the <memory.h> header file. */
+#define HAVE_MEMORY_H 1
+
+/* Define to 1 if you have the <stdint.h> header file. */
+#define HAVE_STDINT_H 1
+
+/* Define to 1 if you have the <stdlib.h> header file. */
+#define HAVE_STDLIB_H 1
+
+/* Define to 1 if you have the <strings.h> header file. */
+#define HAVE_STRINGS_H 1
+
+/* Define to 1 if you have the <string.h> header file. */
+#define HAVE_STRING_H 1
+
+/* Define to 1 if you have the <sys/stat.h> header file. */
+#define HAVE_SYS_STAT_H 1
+
+/* Define to 1 if you have the <sys/types.h> header file. */
+#define HAVE_SYS_TYPES_H 1
+
+/* Define to 1 if you have the <unistd.h> header file. */
+#define HAVE_UNISTD_H 1
+
+/* Define to the sub-directory in which libtool stores uninstalled libraries.
+   */
+#define LT_OBJDIR ".libs/"
+
+/* Define to 1 if your C compiler doesn't accept -c and -o together. */
+/* #undef NO_MINUS_C_MINUS_O */
+
+/* Name of package */
+#define PACKAGE "libogg"
+
+/* Define to the address where bug reports for this package should be sent. */
+#define PACKAGE_BUGREPORT ""
+
+/* Define to the full name of this package. */
+#define PACKAGE_NAME ""
+
+/* Define to the full name and version of this package. */
+#define PACKAGE_STRING ""
+
+/* Define to the one symbol short name of this package. */
+#define PACKAGE_TARNAME ""
+
+/* Define to the home page for this package. */
+#define PACKAGE_URL ""
+
+/* Define to the version of this package. */
+#define PACKAGE_VERSION ""
+
+/* The size of `int', as computed by sizeof. */
+#define SIZEOF_INT 4
+
+/* The size of `int16_t', as computed by sizeof. */
+#define SIZEOF_INT16_T 2
+
+/* The size of `int32_t', as computed by sizeof. */
+#define SIZEOF_INT32_T 4
+
+/* The size of `int64_t', as computed by sizeof. */
+#define SIZEOF_INT64_T 8
+
+/* The size of `long', as computed by sizeof. */
+#define SIZEOF_LONG 4
+
+/* The size of `long long', as computed by sizeof. */
+#define SIZEOF_LONG_LONG 8
+
+/* The size of `short', as computed by sizeof. */
+#define SIZEOF_SHORT 2
+
+/* The size of `uint16_t', as computed by sizeof. */
+#define SIZEOF_UINT16_T 2
+
+/* The size of `uint32_t', as computed by sizeof. */
+#define SIZEOF_UINT32_T 4
+
+/* The size of `u_int16_t', as computed by sizeof. */
+#define SIZEOF_U_INT16_T 2
+
+/* The size of `u_int32_t', as computed by sizeof. */
+#define SIZEOF_U_INT32_T 4
+
+/* Define to 1 if you have the ANSI C header files. */
+#define STDC_HEADERS 1
+
+/* Version number of package */
+#define VERSION "1.3.0"
+
+/* Define to empty if `const' does not conform to ANSI C. */
+/* #undef const */

--- a/deps/libogg/config/linux/arm/ogg/config_types.h
+++ b/deps/libogg/config/linux/arm/ogg/config_types.h
@@ -1,0 +1,25 @@
+#ifndef __CONFIG_TYPES_H__
+#define __CONFIG_TYPES_H__
+
+/* these are filled in by configure */
+#define INCLUDE_INTTYPES_H 1
+#define INCLUDE_STDINT_H 1
+#define INCLUDE_SYS_TYPES_H 1
+
+#if INCLUDE_INTTYPES_H
+#  include <inttypes.h>
+#endif
+#if INCLUDE_STDINT_H
+#  include <stdint.h>
+#endif
+#if INCLUDE_SYS_TYPES_H
+#  include <sys/types.h>
+#endif
+
+typedef short ogg_int16_t;
+typedef unsigned short ogg_uint16_t;
+typedef int ogg_int32_t;
+typedef unsigned int ogg_uint32_t;
+typedef long long ogg_int64_t;
+
+#endif

